common/flagdproxy/flagdproxy.go

```diff
@@ -172,6 +174,9 @@ func (f *FlagdProxyHandler) newFlagdProxyManifest(ownerReference *metav1.OwnerRe
 				},
 				Spec: corev1.PodSpec{
 					ServiceAccountName: FlagdProxyServiceAccountName,
+					ImagePullSecrets: []corev1.LocalObjectReference{
```

Comment:
Should we add it also if it's empty (equals "")?

Reply:
I've done this, thanks @odubajDT